python/xcfun.py

```diff
@@ -11,7 +9,8 @@
 #
 # For information on the complete list of contributors to the
 # XCFun library, see: <https://xcfun.readthedocs.io/>
-#
+
+# -*- coding: utf-8 -*-
```

Comment:
see above (in __init__.py)

Reply:
Done. The pre-commit hook to adjust the license messed it up...